examples/online-exhibition/README.md

```diff
@@ -0,0 +1,168 @@
+
+The directory is organized as follows:
+
+* online-exhibition - contains jobs
+* view.py - contains the code to generate the app
+
+## Scenario
+An association of emerging painters would like to set up an online exhibition on Vincent Van Gogh, a well-known Dutch post-impressionist painter. Within the online exhibition, the association would like to reserve a special section for paintings from the Rijksmuseum in Amsterdam.
+
+The paintings to be included in the exhibition are available on [Europeana](https://www.europeana.eu/en), a well-known European aggregator of the cultural heritage of various kinds. Europeana provides more than 800 works whose author is Vincent Van Gogh.
+
+## Data Source
+
+Europeana provides a [REST API](https://pro.europeana.eu/page/intro#access) for accessing resources. For each resource, various information is available, including the organization that provided the resource. 
+
+For each record, the REST API provides the following information:
+* completeness
+* country varchar,
+* dataProvider varchar,
+* dcCreator varchar,
+* dcCreatorLangAware varchar,
+* dcTitleLangAware varchar,
+* edmDatasetName
+* edmIsShownBy
+* edmPreview
+* edmTimespanLabel
+* edmTimespanLabelLangAware
+* europeanaCollectionName
+* europeanaCompleteness
+* guid
+* id
+* index
+* language
+* link
+* previewNoDistribute
+* provider
+* rights
+* score
+* timestamp
+* timestamp_created
+* timestamp_created_epoch
+* timestamp_update
+* timestamp_update_epoch
+* title
+* type
+* ugc
+
+The following JSON shows a record extract returned by the API:
+
+```
+[{
+    completeness: 10,
+    country: ["Belgium"],
+    dataProvider: ["Catholic University of Leuven"],
+    dcCreator: ["http://data.europeana.eu/agent/base/59832",
+    "Vincent van Gogh"],
+    dcCreatorLangAware: {
+        def: ["http://data.europeana.eu/agent/base/59832"],
+        en:  ["Vincent van Gogh"]
+    },
+    dcDescription: 
+        ["KU Leuven. Glasdia’s kunstgeschiedenis. Université de Louvain, tussen 1839 en 1939. Fotograaf onbekend. Toegevoegde informatie op dia. Stroming/Stijl: Postimpressionisme. Creatie/Bouw: 1889. Techniek/Materiaal: Olieverf op doek. Huidige locatie: Nederland, Otterlo, Kröller-Müller Museum. EuroPhot. Kunstgeschiedenis. 19de eeuw. Schilderkunst. Portret. EuroPhot. Art history. 19th century. Paintings. Portrait."],
+        ...
+}]
+```
+
+The access point to the REST API is available at the following link:
+```
+https://api.europeana.eu/record/v2/search.json
+```
+It is necessary to register for free to obtain an API key.
+
+
+## Requirements
+
+To run this example, you need:
+
+* Versatile Data Kit
+* Trino DB
+* Versatile Data Kit plugin for Trino
+* Europeana API key
+* Streamlit
+
+For more details on how to install Versatile Data Kit, Trino DB, Streamlit and Versatile Data Kit plugin for Trino, please refer to [this link](https://github.com/vmware/versatile-data-kit/tree/main/examples/life-expectancy).
```

Comment:
Do you see any value in creating higher integration between streamlit and vdk? For example adding something like `vdk streamlit-run` which would automatically provide preconfigured connection string, or even the output of some step . 

It has nothing to do with the review. Just thinking if we can improve the experience of users creating dashboards with streamlit with vdk.

Reply:
Yes, I think it could be very useful